File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L6/08/Kasus2L6.py
import math

def main():
    # Enter radius of the cylinder
    print("Enter the radius and length of a cylinder: ", end="")
    radius = float(input())
    # Calculate area
    area = calculate_area(radius)
    # Enter length
    length = float(input())
    # Print area of cylinder
    print("The area is", area)
    # Print volume of cylinder
    print("The volume of the cylinder is", calculate_volume(area, length))

def calculate_area(radius):
    return radius * radius * math.pi

def calculate_volume(area, length):
    return area * length

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-02/plagiarized/L6/08/Kasus2L6.kt
package IR

import java.util.Scanner

fun main(args: Array<String>) {
    // Enter radius of the cylinder
    println("Enter the radius and length of a cylinder: ")
    val scanner = Scanner(System.`in`)
    // Input radius
    val radius = scanner.nextFloat()
    // Calculate area
    val area = calculateArea(radius)
    // Input length
    val length = scanner.nextFloat()
    // Print area of cylinder
    println("The area is $area")
    // Print volume of cylinder
    println("The volume of the cylinder is ${calculateVolume(area, length)}")
}

private fun calculateArea(radius: Float): Double {
    return radius * radius * 3.14159
}

private fun calculateVolume(area: Double, length: Float): Double {
    return area * length
}

